account/src/main.js

```diff
@@ -1,24 +1,22 @@
-import express from 'express';
-import router from './routes.js';
-import swaggerUi from 'swagger-ui-express';
-import swaggerDocs from './swagger.json' assert {type: "json"};
-
-// Importante manter a ordem seguida aqui
+const swaggerUi = require('swagger-ui-express');
+const swaggerDocs = require('./swagger.json');
+const app = require('./app.js');
+const mongoose = require('mongoose');
 
+app.use("/docs", swaggerUi.serve, swaggerUi.setup(swaggerDocs));
 
-// Criando instancia do express
-const app = express();
+require('./controllers/userController')(app);
```

Comment:
i think the right approach is to create a variable with the import before and call the variable after